src/Entity/Midata/GroupType.php

```diff
@@ -16,6 +16,7 @@ class GroupType
     public const CANTON = 'Group::Kantonalverband';
     public const REGION = 'Group::Region';
     public const DEPARTMENT = 'Group::Abteilung';
+    public const DEPARTMENT_HIERARCHY = [self::FEDERATION, self::CANTON, self::REGION, self::DEPARTMENT];
```

Comment:
this naming is not so clear, maybe use something like `DEPARTMENTS_ALLOWING_HIERARCHY`